code/game/objects/structures/crates_lockers/crates/gear_loadout.dm

```diff
@@ -755,6 +755,56 @@
 	new /obj/item/ammo_magazine/c45m(src)
 	new /obj/item/ammo_magazine/c45m(src)
 
+/obj/structure/closet/crate/secure/gear_loadout/ceres_lance/fill()
+	new /obj/item/clothing/under/lance(src)
+	new /obj/item/clothing/under/lance(src)
+	new /obj/item/clothing/under/lance(src)
+	new /obj/item/clothing/under/lance(src)
+	new /obj/item/clothing/under/dress/lance_dress(src)
+	new /obj/item/clothing/under/dress/lance_dress/male(src)
+	new /obj/item/clothing/under/lance(src)
+	new /obj/item/clothing/under/dress/lance_dress(src)
+	new /obj/item/clothing/under/dress/lance_dress/male(src)
+	new /obj/item/clothing/suit/space/void/lancer(src)
+	new /obj/item/clothing/head/helmet/space/void/lancer(src)
+	new /obj/item/clothing/suit/space/void/lancer(src)
+	new /obj/item/clothing/head/helmet/space/void/lancer(src)
+	new /obj/item/clothing/suit/space/void/lancer(src)
+	new /obj/item/clothing/head/helmet/space/void/lancer(src)
+	new /obj/item/clothing/suit/space/void/lancer(src)
+	new /obj/item/clothing/head/helmet/space/void/lancer(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/gun/energy/rifle/ionrifle(src)
+	new /obj/item/gun/projectile/heavysniper(src)
+	new /obj/item/ammo_casing/a145(src)
+	new /obj/item/ammo_casing/a145(src)
+	new /obj/item/ammo_casing/a145(src)
+	new /obj/item/ammo_casing/a145(src)
```

Comment:
No to this too. This weapon is out of gameplay for good reason too; it's an instant-kill through walls. Even worse with thermals.

Make it a scout rifle or a PEAC.